### ISSUE TITLE
Build on Debian failed

### ISSUE THREAD

**Issue by pavroo** [BlankOn/manokwari#103]
Hi
I build debs on Debian stable and testing, and there was no problem up to version 1.0.16.
After that all build fails.
Checked out, the problem is with installation files of 'files' directory.
All building files from other dirs are installed in $WORKDIR/debian/manokwari/$PREFIX dirs, but the files from the 'files' directory want to install in the system tree, such as /usr/bin, /etc/xdg/menus, but not in the $WORKDIR. 
Can you fix it, please?

**Comment by antosamalona** [comment]
Yes, BlankOn developers still working on it. Meanwhile you can use Makefile.am 1.0.16 version.

**Comment by alunux** [comment]
@pavroo if you know how to build stuffs using meson, you should try it.
